lib/node_modules/@stdlib/math/base/special/floorn/src/floorn.c

```diff
@@ -32,8 +32,9 @@
 
 // VARIABLES //
 
+#define HUGE 1.0e308
```

Comment:
Same comment as above.

Reply:
We need to be careful using `define`, as you don't always know what symbols are imported through via `#includes`. You could end up clobbering an existing macro, which is why we'd need to namespace and add include guards. The static const declaration avoids this.